### PR TITLE
modulesync 0.12.2

### PR DESCRIPTION
**Pull request by bastelfreak**
this got cherry-picked from https://github.com/voxpupuli/puppet-zabbix/pull/268

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ notifications:
   email: false
 deploy:
   provider: puppetforge
-  edge:
```
Comment (bastelfreak) [inline .travis.yml]: no clue why we had edge here. We successfully tested it with deploy.

**Comment by bastelfreak** [comment]
We've to wait until https://github.com/stompgem/stomp/pull/133 is merged + released. This currently breaks our travis.

**Comment by gmallard** [comment]
The difficulties described in https://github.com/stompgem/stomp/pull/133 have been fixed with stomp gem 1.4.3 which is now available on rubygems.

Sorry for the hassle.

Regards, Guy

**Comment by bastelfreak** [comment]
Hi @gmallard, thanks for the fast response!

**Pull request merged**